packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -48,7 +48,8 @@ export const ModalButton = ({
       className={classNames(
         'cta-base transition-standard',
         leftSpaced && 'ml-2',
-        className || 'cta-neutral'
+        className || 'cta-neutral',
+        'py-2 px-5'
```

Comment:
I think this would apply `py-2 px-5` no matter what, I think this would need to be `className || cta-neutral py-2 px-5` instead of `className || 'cta-neutral', 'py-2 px-5'`.

Reply:
Oh good thinking